app/jinja_filters.py

```diff
@@ -153,11 +153,15 @@ def format_date_custom(context, value, date_format='EEEE d MMMM YYYY'):
 
 @evalcontextfilter
 @blueprint.app_template_filter()
-def format_datetime(context, value, date_format="d MMMM YYYY 'at' HH:mm"):
+def format_datetime(context, value):
 
     london_date_time = datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f')
-    result = "<span class='date'>{date}</span>".format(date=flask_babel.format_datetime(london_date_time,
-                                                                                        format=date_format))
+    formatted_date = flask_babel.format_date(london_date_time, format='d MMMM YYYY')
+    formatted_time = flask_babel.format_time(london_date_time, format='HH:mm')
+
+    result = "<span class='date'>{date}</span>".format(
```

Comment:
Not sure if this is in the scope of this PR, but hardcoding "_date_ at _time_" won't translate.  
It's not enough to block this PR because it's how it was beforehand, but this should probably be addressed somewhere